crm_claim_rma/models/account_invoice.py

```diff
@@ -13,47 +13,53 @@ class AccountInvoice(models.Model):
 
     claim_id = fields.Many2one('crm.claim', string='Claim')
 
+    @api.model
     def _refund_cleanup_lines(self, lines):
-        """ Override when from claim to update the quantity and link to the
-        claim line.
+        """ Override when from claim to update the quantity, link to the
+        claim line and return empty list for tax_line_ids.
         """
+        claim_lines_context = self._context['claim_line_ids']
 
-        # check if is an invoice_line and we are from a claim
-        if not (self.env.context.get('claim_line_ids') and lines and
-                lines[0]._name == 'account.invoice.line'):
```

Comment:
@kkarolis This method is used twice at the moment - [here](https://github.com/odoo/odoo/blob/91807404e7d5c314ea30926b44bd16e878417ec1/addons/account/models/account_invoice.py#L1066). Currently tax_line_ids are being appended to new invoice. In case of full refund (odoo default), this is ok. In case of claim that has only few lines - taxes are incorrect, therefore we are returning empty list for `account.invoice.tax` .